bin/demultiplexingTiming.sh

```diff
@@ -172,54 +173,49 @@ do
     fi
 
 
-    timeStamp=$(find "${SCR_ROOT_DIR}/logs/${sequenceRun}/" -type f -mmin +60 -iname "run0"[1-9]"demultiplexing.started")
-    echo "timestamp check"
-    if [[ -z "${timeStamp}" ]]
+    if [ -e "${SCR_ROOT_DIR}/logs/${sequenceRun}/${run}.demultiplexing.finished" ]
     then
-        echo "timestamp is niet leeg!"
-        if [ -e "${SCR_ROOT_DIR}/logs/${sequenceRun}/run0"[1-9]".demultiplexing.finished" ]
+        echo ".finished bestaat"
```

Comment:
is dit een debug echo??

Reply:
ja, was zo makkelijker voor mezelf om te zien hoe het script verliep, mag er dus uit?